tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -4,30 +4,31 @@
 from ocs_ci.ocs import constants
 from ocs_ci.ocs.ocp import OCP
 from ocs_ci.ocs.resources import storage_cluster
-from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.utility.utils import ceph_health_check
 
 
 @ignore_leftovers
 @tier1
 @polarion_id('OCS-1191')
 class TestAddCapacity(ManageTest):
     """
+
     Automates adding variable capacity to the cluster while IOs running
     """
     def test_add_capacity(self):
         """
-        Test to add variable capacity to the OSD cluster while IOs running
-        """
-        self.ceph_cluster = CephCluster()
+       Test to add variable capacity to the OSD cluster while IOs running
+       """
+        assert ceph_health_check(), 'health check failed'
```

Comment:
if the intention is to check Ceph health as a prerequisite for the test to start, better to do it in the test setup.

like done here https://github.com/red-hat-storage/ocs-ci/blob/6f9a193c585ffd5770d840466db14182a906f49e/tests/manage/z_cluster/nodes/test_nodes_maintenance.py#L58

Reply:
Thanks, will do